### ISSUE TITLE
SlowArrayOperationsInLoopInspector dont find all array merge 

### ISSUE THREAD

**Issue by larsroettig** [kalessil/phpinspectionsea#1253]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Ultimate) |
| Language level | PHP 7.2                        |

Current behaviour
--

```php
class CSV
{
    public static function getColumns()
    {
        return [];
    }
}

foreach ([] as $collection) {
    foreach ($configurationSources as $source) {
        $options = array_merge(CSV::getColumns(), $source);
    }
}
```
Expected behaviour
--

He should find the array_merge 

Environment details
---

**Comment by kalessil** [comment]
So, this specific case I considering legit as merge is not involving `$options`.
On other hand we are reporting `$object->setArray(array_merge($object->getArray(), ...)` now.
